collect_app/src/main/res/layout/storage_migration_banner.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/storageMigrationBanner"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="?colorSurface"
+    android:visibility="gone">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:paddingStart="@dimen/margin_standard"
+        android:paddingEnd="@dimen/margin_standard"
+        android:paddingTop="@dimen/margin_standard">
+
+        <TextView
+            android:id="@+id/storageMigrationBannerText"
```

Comment:
I don't think so then it would be like:
![Screenshot_2020-02-21-09-19-12](https://user-images.githubusercontent.com/3276264/75014672-25388280-5487-11ea-99a2-b94fc5af2c97.png)
but it should be like:
![image](https://user-images.githubusercontent.com/3276264/75014701-32ee0800-5487-11ea-811d-66abe215cc46.png)

Reply:
@grzesiek2010 are you sure that's `Body2`? It looks like it might be the `Button` style.

Reply:
Right i did something wrong. Fixed.